fotobox/download-app/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM  golang:1.23
```

Comment:
please don't use a dockerhub image as base image.

I suggest to take a look at the following dockerfile that gives you a working example that builds a scratch image